vagrant/boxes.d/00-ubuntu.yaml

```diff
@@ -33,3 +33,23 @@ boxes:
     ansible:
       variables:
         ansible_python_interpreter: /usr/bin/python3
+  ubuntu2204:
+    box_name: 'generic/ubuntu2204'
+    pty: true
+    scenarios:
+      - foreman
+    shell: |
+      # Work around external nameservers in generic/ubuntu2204
+      # https://github.com/lavabit/robox/issues/11
+      sed -i 's/\(addresses:\).\+/\1 []/' /etc/netplan/01-netcfg.yaml
+      # Work around netplan configs having too open permissions
+      # https://github.com/lavabit/robox/issues/296
+      chmod 600 /etc/netplan/*.yaml
+      netplan generate
+      netplan apply
+      sed -i 's/^\(DNS=\)/#\1/' /etc/systemd/resolved.conf
+      sed -i 's/^DNSSEC=.*/DNSSEC=no/' /etc/systemd/resolved.conf
+      systemctl restart systemd-resolved
+    ansible:
+      variables:
+        ansible_python_interpreter: /usr/bin/python3
```

Comment:
Is this still needed? I think current Ansible can figure this out.

Reply:
Probably not, I'll check and remove before merging :)